beacon_api/conf/__init__.py

```diff
@@ -58,3 +59,6 @@ def parse_oauth2_config_file(path):
 
 
 OAUTH2_CONFIG = parse_oauth2_config_file(os.environ.get('CONFIG_FILE', os.path.join(os.path.dirname(__file__), 'config.ini')))
+
```

Comment:
```suggestion
# Sample query file should be of format [{BeaconAlleleRequest}] https://github.com/ga4gh-beacon/specification/
```

Reply:
Maybe also add a comment that indicates the structure in the `sample_queries.json`?